includes/Extension/SearchReplace/class-replace.php

```diff
@@ -85,12 +96,163 @@ public function add_replace_button() {
 		);
 	}
 
+	/**
+	 * Output a button to display the create staging site.
+	 *
+	 * @return void
+	 */
+	public function add_instawp_stage_button() {
+		self::install_plugin_button( 'instawp-connect', 'instawp-connect.php', 'InstaWP Connect', array(), __( 'Go to InstaWP &rarr;', 'string-locator' ), __( 'Activate InstaWP', 'string-locator' ), __( 'Create a Staging Site (powered by InstaWP)', 'string-locator' ) );
+	}
+
+	public static function install_plugin_button( $plugin_slug, $plugin_file, $plugin_name, $classes = array(), $activated = '', $activate = '', $install = '' ) {
+		if ( current_user_can( 'install_plugins' ) && current_user_can( 'activate_plugins' ) ) {
+			if ( is_plugin_active( $plugin_slug . '/' . $plugin_file ) ) {
+				// The plugin is already active.
+				// $instawp_connect = menu_page_url( 'instawp-connect', false );				
+				$instawp_connect = menu_page_url( 'instawp', false );				
+				$button = array(
+					'message' => esc_attr__( 'Create a Staging Site', 'string-locator' ),
+					'url'     => $instawp_connect,
+					'classes' => array( 'string-locator-instawp-button', 'disabled' ),
+				);
+
+				if ( '' !== $activated ) {
+					$button['message'] = esc_attr( $activated );
+
+					$button['target'] = 'target="_blank"';
+				}
+				$button['logo-img'] = '<span class="btn-logo-inline"><img src="'. plugins_url( 'views/assets/instawp-logo.svg', STRING_LOCATOR_PLUGIN_FILE ).'" alt="InstaWP logo"></span>';
```

Comment:
The image source is not escaping the URL here.

Reply:
Reolved in latest commit.